.circleci/config.yml

```diff
@@ -384,37 +375,22 @@ workflows:
               only:
                 - development
     jobs:
-      - build
-      - coding_standards:
-          requires:
-            - build
-      - deprecated_code:
-          requires:
-            - build
-      - unit_kernel_tests:
```

Comment:
Banished this long running, much despised (by me!) step, as our build integrity is going to be verified nightly in our edge build process.

There's a chance something might break and we won't know until we try to build it... but the edge build is volatile anyway so if we need to sit, fix + push it up to rebuild it, then so be it. By shifting this step into the edge build process, we speed up routine code commits and PRs.